pkg/util/migrations/migrations.go

```diff
@@ -112,3 +111,21 @@ func VMIMigratableOnEviction(clusterConfig *virtconfig.ClusterConfig, vmi *v1.Vi
 	}
 	return false
 }
+
+func InterruptedMigrationForVMI(migrationIndexer cache.Indexer, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstanceMigration, error) {
```

Comment:
Can we use the store object as it is the case for [ListUnfinishedMigrations](https://github.com/kubevirt/kubevirt/blob/main/pkg/util/migrations/migrations.go#L16) or [ListWorkloadUpdateMigrations](https://github.com/kubevirt/kubevirt/blob/main/pkg/util/migrations/migrations.go#L28)

Reply:
Why? The indexer allows for a cleaner namespace filter...

Reply:
ACK!